src/components/button-toggle/button-toggle.stories.mdx

```diff
@@ -18,155 +20,161 @@ Press one of the buttons to make selection. ButtonToggle component should be use
 
 ## Quick Start
 
-To use the `ButtonToggle` component you have to import the `<ButtonToggle />` component:
-
 ```javascript
 import ButtonToggle from "carbon-react/lib/components/button-toggle";
+import ButtonToggleGroup from "carbon-react/lib/components/button-toggle-group";
 ```
 
 ## Designer Notes
 
+- If you need to wrap multiple `ButtonToggle` components together, **use the <LinkTo kind="Button Toggle Group" story="default">ButtonToggleGroup component</LinkTo> only.**
```

Comment:
I'd remove the word "only", as this reads strangely to me. (Maybe it's just me though?)

If we need to emphasise that this is the only approved way to do it, I'd instead phrase it as, `If you need to wrap multiple ButtonToggle components together, you should/must use the ButtonToggleGroup component`